modules/file-system/pre-existing-network-storage/outputs.tf

```diff
@@ -24,3 +24,55 @@ output "network_storage" {
     mount_options = var.mount_options
   }
 }
+
+locals {
+  # Client Install
+  ddn_lustre_client_install_script = templatefile(
+    "${path.module}/templates/ddn_exascaler_luster_client_install.tftpl",
+    {
+      server_ip    = split("@", var.server_ip)[0]
+      remote_mount = var.remote_mount
+      local_mount  = var.local_mount
+    }
+  )
+
+  install_scripts = {
```

Comment:
Just to clarify my understanding, the idea is that eventually all mount/install scripts and templates will be copied here (just nfs for now really) and automatically generated if the fs_type is the same?

Reply:
Yes.